src/Acuminator/Acuminator.Utilities/DiagnosticSuppression/SuppressionManager.cs

```diff
@@ -88,53 +83,82 @@ public static void Init(ISuppressionFileSystemService fileSystemService, IEnumer
 
 		public static void SaveSuppressionBase()
 		{
-			if (Instance == null)
+			CheckIfInstanceIsInitialized();
+			
+			lock (Instance._fileSystemService)
```

Comment:
@SENya1990 why do we need a look here?

Reply:
@metallmonster because the VS throws exception about access to file from another thread.
So I added exclusive access to all places where the file service is used to access the file